source/Halibut.Tests/Util/SerilogLoggerBuilder.cs

```diff
@@ -1,19 +1,68 @@
+using System;
+using System.IO;
+using NUnit.Framework;
 using Serilog;
+using Serilog.Core;
+using Serilog.Events;
 using Serilog.Formatting.Display;
 using Serilog.Sinks.NUnit;
 
 namespace Halibut.Tests.Util
 {
     public class SerilogLoggerBuilder
     {
-        const string OutputTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception}";
+        const string OutputTemplate = "{Timestamp:HH:mm:ss.fff zzz} {Message}{NewLine}{Exception}";
```

Comment:
I'm being blind I think, but we've updated this private const, and it looks like we aren't using it below.

Reply:
good point let me fix that up.